explore/src/main/scala/explore/config/SequenceEditorPopup.scala

```diff
@@ -36,41 +35,38 @@ object SequenceEditorPopup:
   private val component =
     ScalaFnComponent
       .withHooks[Props]
-      .useState(false)        // isOpen
+      .useStateView(false)    // isOpen
```

Comment:
I've been trying to switch these to use `NewTypes` to make them more readable

Reply:
Sure, I can switch those in the morning.